package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "scholarxbackend",
```

Comment:
Shall we change this to scholarx-backend? I think it looks better that way.
```suggestion
  "name": "scholarx-backend",
```

Reply:
Run the npm install command once, so it will update the name on the package-lock.json as well.

Reply:
Done